### ISSUE TITLE
Move to AndroidX

### ISSUE THREAD

**Issue by XIZZW** [wirecube/android_additive_animations#16]
Would you consider move to AndroidX? I still get the following running `canISayByeByeJetifier`

```
Scanning at.wirecube:additive_animations:1.8.0
 Absoulute path: /Users/xiwei/.gradle/caches/modules-2/files-2.1/at.wirecube/additive_animations/1.8.0/1af778f898822b9b74d83f8c3632188f191fc41b/additive_animations-1.8.0.aar
 Graphs to this dependency:
 +---at.wirecube:additive_animations:1.8.0
 Issues found:
 * at/wirecube/additiveanimations/additive_animator/SubclassableAdditiveViewAnimator.class -> android/support/v4/view/ViewCompat
 * at/wirecube/additiveanimations/helper/EaseInOutPathInterpolator.class -> android/support/v4/view/animation/PathInterpolatorCompat
```

Thanks

**Comment by davidganster** [comment]
Hi, 

thanks for bringing this up. 
Yes, I will migrate to AndroidX and release an updated version.
I also plan on eventually migrating to Kotlin to take advantage of some Kotlin-specific language features!

**Comment by davidganster** [comment]
I just moved the project to use AndroidX and released v1.9.0.

Cheers,
David

**Comment by XIZZW** [comment]
Thank you!
